### PR TITLE
Rename xenondb to randondb-mysql  #51

### PR DESCRIPTION
**Pull request by molliezhang**
1. Modify README.md 
2. Modify deploy files in docs

### CODE REVIEW

File: docs/KubeSphere/deploy_randondb-mysql_on_kubesphere.md [hunk 1]
```diff
@@ -234,7 +234,7 @@ mysql -h my-release-xenondb-follower -u <用户名> -p<密码>
 | `replicaCount`                               | Pod 数目                                                 | `3`                                     |
 | `busybox.image`                              | `busybox` 镜像库地址                                       | `busybox`                               |
 | `busybox.tag`                                | `busybox` 镜像标签                                        | `1.32`                                   |
-| `mysql.image`                                | `mysql` 镜像库地址                                         | `xenondb/percona`                     |
+| `mysql.image`                                | `mysql` 镜像库地址                                         | `radondb-mysql/percona`                     |
```
Comment (andyli029) [inline docs/KubeSphere/deploy_randondb-mysql_on_kubesphere.md:237]: not rename here

File: docs/KubeSphere/deploy_randondb-mysql_on_kubesphere.md [hunk 2]
```diff
@@ -34,25 +34,25 @@ XenonDB 是基于 MySQL 的开源、高可用、云原生集群解决方案。
...
-#### **步骤 1：克隆 XenonDB Chart**
+#### **步骤 1：克隆 RadonDB-MySQL Chart**
 
 1. 通过第三方 SSH 客户端，连接 KubeSphere 客户端节点。
-2. 执行如下命令，将 XenonDB Chart 克隆到 KubeSphere 客户端节点中。
+2. 执行如下命令，将 RadonDB-MySQL Chart 克隆到 KubeSphere 客户端节点中。
 
    ```bash
-   git clone https://github.com/radondb/xenondb.git
+   git clone https://github.com/radondb/radondb-mysql.git
```
Comment (zhyass) [inline docs/KubeSphere/deploy_randondb-mysql_on_kubesphere.md:47]: radondb-mysql-kubernetes

File: docs/KubeSphere/deploy_randondb-mysql_on_kubesphere.md [hunk 3]
```diff
@@ -97,24 +97,24 @@ XenonDB 是基于 MySQL 的开源、高可用、云原生集群解决方案。
...
 
-#### **步骤 1：创建 XenonDB Chart 压缩包**
+#### **步骤 1：创建 RadonDB-MySQL Chart 压缩包**
 
-拉取 XenonDB Chart。
+拉取 RadonDB-MySQL Chart。
 
 ```bash
-git clone https://github.com/radondb/xenondb.git
+git clone https://github.com/radondb/radondb-mysql.git
```
Comment (zhyass) [inline docs/KubeSphere/deploy_randondb-mysql_on_kubesphere.md:111]: radondb-mysql-kubernetes

File: docs/KubeSphere/deploy_randondb-mysql_on_kubesphere.md [hunk 4]
```diff
@@ -97,24 +97,24 @@ XenonDB 是基于 MySQL 的开源、高可用、云原生集群解决方案。
...
 ```
 
 打包生成 tgz 或 tar.gz 文件。
 
 ```bash
-cd xenondb
+cd radondb-mysql
```
Comment (zhyass) [inline docs/KubeSphere/deploy_randondb-mysql_on_kubesphere.md:117]: radondb-mysql-kubernetes

File: docs/KubeSphere/deploy_randondb-mysql_on_kubesphere.md (hunk 1 above)
Comment (zhyass) [inline docs/KubeSphere/deploy_randondb-mysql_on_kubesphere.md:237]: Donot modify the repository

File: docs/KubeSphere/deploy_randondb-mysql_on_kubesphere.md [hunk 5]
```diff
@@ -257,7 +257,7 @@ mysql -h my-release-xenondb-follower -u <用户名> -p<密码>
 | `mysql.readinessProbe.failureThreshold`      | 就绪探测失败的重试次数，重试一定次数后将认为容器未就绪              | 3                                      |
 | `mysql.extraEnvVars`                         | 其他作为字符串传递给 `tpl` 函数的环境变量                       |                                         |
 | `mysql.resources`                            | `MySQL` 的资源请求/限制                                      | 内存: `256Mi`, CPU: `100m`              |
-| `xenon.image`                                | `xenon` 镜像库地址                                          | `xenondb/xenon`                       |
+| `xenon.image`                                | `xenon` 镜像库地址                                          | `radondb-mysql/xenon`                       |
```
Comment (zhyass) [inline docs/KubeSphere/deploy_randondb-mysql_on_kubesphere.md:260]: Donot modify the repository

File: docs/Kubernetes/deploy_randondb-mysql_on_kubernetes.md [hunk 6]
```diff
@@ -1,30 +1,30 @@
...
 
-### **步骤 1：克隆 XenonDB Chart**
+### **步骤 1：克隆 RadonDB-MySQL Chart**
 
-执行如下命令，将 XenonDB Chart 克隆到 Kubernetes 中。
+执行如下命令，将 RadonDB-MySQL Chart 克隆到 Kubernetes 中。
 
 ```bash
-git clone https://github.com/radondb/xenondb.git
+git clone https://github.com/radondb/radondb-mysql.git
```
Comment (zhyass) [inline docs/Kubernetes/deploy_randondb-mysql_on_kubernetes.md:20]: radondb-mysql-kubernetes

File: docs/Kubernetes/deploy_randondb-mysql_on_kubernetes.md [hunk 7]
```diff
@@ -1,30 +1,30 @@
...
 ```
 
 > Chart 代表 [Helm](https://helm.sh/zh/docs/intro/using_helm/) 包，包含在 Kubernetes 集群内部运行应用程序、工具或服务所需的所有资源定义。
 
 ### **步骤 2：部署**
 
-在 xenondb 目录路径下，选择如下方式，部署 release 实例。
+在 RadonDB-MySQL 目录路径下，选择如下方式，部署 release 实例。
```
Comment (zhyass) [inline docs/Kubernetes/deploy_randondb-mysql_on_kubernetes.md:27]: radondb-mysql-kubernetes 目录

File: docs/KubeSphere/deploy_randondb-mysql_on_kubesphere.md [hunk 8]
```diff
@@ -34,25 +34,25 @@ XenonDB 是基于 MySQL 的开源、高可用、云原生集群解决方案。
...
+   git clone https://github.com/radondb/radondb-mysql.git
    ```
 
 > Chart 代表 [Helm](https://helm.sh/zh/docs/intro/using_helm/) 包，包含在 Kubernetes 集群内部运行应用程序、工具或服务所需的所有资源定义。
 
 #### **步骤 2：部署**
 
 1. 通过第三方 SSH 客户端，连接 KubeSphere 客户端节点。
-2. 在 xenondb 目录路径下，选择如下方式，部署 release 实例。
+2. 在 RadonDB-MySQL 目录路径下，选择如下方式，部署 release 实例。
```
Comment (zhyass) [inline docs/KubeSphere/deploy_randondb-mysql_on_kubesphere.md:55]: radondb-mysql-kubernetes 目录

File: docs/Kubernetes/deploy_randondb-mysql_on_kubernetes.md [hunk 9]
```diff
@@ -131,7 +131,7 @@ kubectl get statefulset,pod,svc
 | `replicaCount`                               | Pod 数目                                                 | `3`                                     |
 | `busybox.image`                              | `busybox` 镜像库地址                                       | `busybox`                               |
 | `busybox.tag`                                | `busybox` 镜像标签                                        | `1.32`                                   |
-| `mysql.image`                                | `mysql` 镜像库地址                                         | `xenondb/percona`                     |
+| `mysql.image`                                | `mysql` 镜像库地址                                         | `radondb-mysql/percona`                     |
```
Comment (zhyass) [inline docs/Kubernetes/deploy_randondb-mysql_on_kubernetes.md:134]: Donot modify the repository

File: docs/Kubernetes/deploy_randondb-mysql_on_kubernetes.md [hunk 10]
```diff
@@ -154,7 +154,7 @@ kubectl get statefulset,pod,svc
 | `mysql.readinessProbe.failureThreshold`      | 就绪探测失败的重试次数，重试一定次数后将认为容器未就绪              | 3                                      |
 | `mysql.extraEnvVars`                         | 其他作为字符串传递给 `tpl` 函数的环境变量                       |                                         |
 | `mysql.resources`                            | `MySQL` 的资源请求/限制                                      | 内存: `256Mi`, CPU: `100m`              |
-| `xenon.image`                                | `xenon` 镜像库地址                                          | `xenondb/xenon`                       |
+| `xenon.image`                                | `xenon` 镜像库地址                                          | `radondb-mysql/xenon`                       |
```
Comment (zhyass) [inline docs/Kubernetes/deploy_randondb-mysql_on_kubernetes.md:157]: Donot modify the repository

**Review by zhyass — CHANGES_REQUESTED** [review]

**Comment by zhyass** [comment]
Please link to this issue #75 
Modify the commit information as `*: Modify README.md anddeployfiless #75`

**Pull request closed without merging**